Sources/EpoxyCore/SwiftUI/EpoxySwiftUIHostingView.swift

```diff
@@ -71,6 +71,7 @@ public final class EpoxySwiftUIHostingView<RootView: View>: UIView, EpoxyableVie
 
     super.init(frame: .zero)
 
+    epoxyEnvironment.hostingController = viewController
```

Comment:
Were you able to verify that this isn't a retain cycle? I'm a bit worried that the environment is strongly retaining the hosting controller and the hosting controller is strongly retaining the environment—wonder if this needs to be a `weak var`

Reply:
See the discussion here as a follow up https://github.com/airbnb/epoxy-ios/pull/83#:~:text=commented-,11%20seconds%20ago,-Closing%20this%20in